apps/src/sites/studio/pages/components/LevelToken.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {Motion, spring} from 'react-motion';
+import color from '../../../../util/color';
+import { borderRadius, levelTokenMargin } from './constants';
+import LevelTokenDetails from './LevelTokenDetails';
+
+const styles = {
+  levelToken: {
+    position: 'relative',
+    fontSize: 13,
+    background: '#eee',
+    borderRadius: borderRadius,
+    margin: `${levelTokenMargin}px 0`
+  },
+  reorder: {
+    fontSize: 16,
+    display: 'table-cell',
+    background: '#ddd',
+    border: '1px solid #bbb',
+    boxShadow: 'inset 0 1px 0 0 rgba(255, 255, 255, 0.6)',
+    padding: '7px 15px',
+    borderTopLeftRadius: borderRadius,
+    borderBottomLeftRadius: borderRadius,
+    cursor: 'ns-resize'
+  },
+  levelTokenName: {
+    padding: 7,
+    display: 'table-cell',
+    boxShadow: 'inset 0 1px 0 0 rgba(255, 255, 255, 0.8)',
+    width: '100%',
+    borderTop: '1px solid #ddd',
+    borderBottom: '1px solid #ddd',
+    cursor: 'text'
+  },
+  variants: {
+    color: 'white',
+    background: color.purple,
+    padding: '3px 5px',
+    lineHeight: '12px',
+    borderRadius: 5,
+    float: 'right'
+  },
+  remove: {
+    fontSize: 14,
+    display: 'table-cell',
+    color: 'white',
+    background: '#c00',
+    border: '1px solid #a00',
+    boxShadow: 'inset 0 1px 0 0 rgba(255, 255, 255, 0.6)',
+    padding: '7px 13px',
+    borderTopRightRadius: borderRadius,
+    borderBottomRightRadius: borderRadius,
+    cursor: 'pointer'
+  }
+};
+
+const LevelToken = React.createClass({
```

Comment:
What is a LevelToken? I'm not always great about this either, but I think would be good for us to have class level comments for most of our components describing what they are.